<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeAddUserDlg</class>
 <widget class="QDialog" name="DeAddUserDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeAddUserDlg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DeAddUserDlg
{
   	background-color:#E9E9E9;
}

QLineEdit
{
border:1px solid gray;
border-radius:5px;
min-height:30px;
max-height:30px;
font: 14px &quot;微软雅黑&quot;;
}

QLabel
{
    font: 14px &quot;微软雅黑&quot;;
	color: #95989A;
}

QGroupBox {
     border: 1px solid gray;
     border-radius: 3px;
     margin-top: 2ex; /* leave space at the top for the title */
     padding: 10px;
     color: #95989A;
     font: 14px &quot;微软雅黑&quot;;
 }

 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 10px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #FFOECE, stop: 1 #FFFFFF);
 }

QPushButton 
{    
	font: bold 14px &quot;微软雅黑&quot;;
	background-color: rgb(244, 244, 244);
    border:none;
    border-radius: 5px;
    min-width: 100px;
    min-height: 30px;
 }

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>200</number>
   </property>
   <property name="topMargin">
    <number>160</number>
   </property>
   <property name="rightMargin">
    <number>200</number>
   </property>
   <property name="bottomMargin">
    <number>160</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <item row="0" column="0" colspan="4">
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>添加用户</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="verticalSpacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>用户名</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="userLineEdit">
           <property name="maxLength">
            <number>32</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>密   码</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="passwdLineEdit">
           <property name="maxLength">
            <number>32</number>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>权  限:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="DeComboBox" name="widgePermission" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>37</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="okBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton 
{
	background-color: #42BDAA;
    color: #FFFFFF;
}</string>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="cancelBtn">
        <property name="text">
         <string>取消</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>37</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DeComboBox</class>
   <extends>QWidget</extends>
   <header location="global">decombobox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
